test/unit/services/test_image.py

```diff
@@ -151,14 +153,24 @@ class TestCreate(CreateDSL):
     def test_create(self):
         """Test creating an image."""
 
-        self.mock_create(IMAGE_POST_METADATA_DATA_ALL_VALUES)
+        self.mock_create(IMAGE_POST_METADATA_DATA_ALL_VALUES, "test.png")
         self.call_create()
         self.check_create_success()
 
+    def test_create_with_invalid_file(self):
```

Comment:
```suggestion
    def test_create_with_file_extension_content_type_mismatch(self):
```
I think it could do with an e2e test too if possible.